legateboost/models/tree.py

```diff
@@ -74,10 +96,12 @@ def fit(
         task.add_scalar_arg(self.max_depth, types.int32)
         max_nodes = 2 ** (self.max_depth + 1)
         task.add_scalar_arg(max_nodes, types.int32)
-        task.add_scalar_arg(self.alpha, types.float64)
+        task.add_scalar_arg(self.l2_regularization, types.float64)
```

Comment:
Can you move this down to be before the l1 regularization?  (or move the l1 up here)